db/migrate/20221116092629_update_tags_and_tabs_for_cms_resources.rb

```diff
@@ -0,0 +1,55 @@
+OLD_TAB_LABELS = ['National resources', 'Global publications', 'Governance documents']
```

Comment:
this looks good but rather than put this into a migration I would move this into a rake task that we can run on the server and potentially reuse in the future (with ad hoc changes). 
The migrations are normally reserved to change the schema of the db rather than the content

Reply:
Popped the migrations back in and rolled back (one of the migrations was already in develop hence why there is a change in the schema in this PR still).